framework_tvb/tvb/core/services/backend_clients/cluster_scheduler_client.py

```diff
@@ -65,9 +65,8 @@ def _run_cluster_job(operation_identifier, user_name_label, adapter_instance):
         """
         # Load operation so we can estimate the execution time
         operation = dao.get_operation_by_id(operation_identifier)
-        kwargs = parse_json_parameters(operation.parameters)
         # kwargs = adapter_instance.prepare_ui_inputs(kwargs)
-        time_estimate = int(adapter_instance.get_execution_time_approximation(**kwargs))
+        time_estimate = int(adapter_instance.get_execution_time_approximation())
```

Comment:
`ABCAdapter.get_execution_time_approximation` expects a loaded `ViewModel` param for correct evaluation.
We should not pass empty here